spec/rails_helper.rb

```diff
@@ -97,3 +97,6 @@
 Capybara.configure do |config|
   config.javascript_driver = :chrome_headless # This is slower
 end
+
+# Disable CSS animations which slows down tests
+Capybara.disable_animation = true
```

Comment:
Nice! 🚀

Reply:
isn't it? also read that css animations and transitions might be causing flakey tests so I tried disabling them globally, it also let to get rid of those `sleep 1` in the other specs as well.

Reply:
Mayyybeee this could fix the flaky spec in myaccount? 🤞 👀

Reply:
yes we should try this over there 🤞🏻